x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/job_map_title.tsx

```diff
@@ -18,13 +18,13 @@ export const JobMapTitle: FC<{ analyticsId?: string; modelId?: string }> = ({
       {analyticsId ? (
```

Comment:
Could the ID just be shown in the page header? So here, the header would be `Map for job ID <id>` , or `Map for trained model ID <id>` if coming in from a model, and we could lose the title inside the map?

![image](https://user-images.githubusercontent.com/7405507/157279019-efdcfffe-ad56-42ea-b3de-fb9dbb066f7b.png)